src/analysis/souffle/BUILD

```diff
@@ -38,14 +38,14 @@ gen_souffle_cxx_code(
 gen_souffle_cxx_code(
     name = "taint_cxx_for_test",
     src = "taint.dl",
+    for_test = True,
```

Comment:
what is this?

Reply:
`buildifier` just moved `for_test` from the end of the rule to the beginning. In an earlier PR, in which I split `gen_souffle_cxx_code` out from `souffle_cc_library`, I replaced `testonly` with `for_test` in the rule parameters, as this new parameter simultaneously sets `testonly` on subrules and sets a conditional compilation parameter to include or not include Datalog used only for tests.